pennylane/resource/specs.py

```diff
@@ -168,24 +399,26 @@ def circuit(x, add_ry=True):
             qml.TrotterProduct(Hamiltonian, time=1.0, n=4, order=4)
             return qml.probs(wires=(0,1))
 
-    >>> print(qml.specs(circuit)(x, add_ry=False))
-    Device: default.qubit
-    Device wires: 2
-    Shots: Shots(total=None)
-    Level: gradient
-    <BLANKLINE>
-    Resource specifications:
-      Total qubit allocations: 2
-      Total gates: 98
-      Circuit depth: 98
-    <BLANKLINE>
-      Gate types:
-        RX: 1
-        CNOT: 1
-        Evolution: 96
-    <BLANKLINE>
-      Measurements:
-        probs(all wires): 1
+    ::
+
+        >>> print(qml.specs(circuit)(x, add_ry=False))
+        Device: default.qubit
+        Device wires: 2
+        Shots: Shots(total=None)
+        Level: gradient
+        <BLANKLINE>
+        Resource specifications:
+          Total qubit allocations: 2
```

Comment:
I think we should stick with `wires` instead of `qubits` :heh:. We use both in the specs output

Reply:
Also, is `Total qubit allocations` coming from `qml.allocate`?

Reply:
499a2873a